azure-pipelines.yml

```diff
@@ -311,3 +311,21 @@ jobs:
   - bash: ci/run_win.sh
     displayName: 'Testing'
 
+- job: PyPi
+  dependsOn:
+    - 'Ubuntu16Py37'
+    - 'Ubuntu16All'
+    - 'Ubuntu16Py27'
+    - 'Ubuntu16PyPy2'
+    - 'Ubuntu16PyPy3'
+    - 'OSX10_13'
+    - 'Win2016'
+  condition: and(succeeded(), eq(variables['build.sourceBranch'], 'refs/heads/master'))
+  steps:
+  - script: 'pip install twine wheel'
+  - script: './make_distrib.sh; ls dist/'
```

Comment:
Inside this script we use `python`, if a specific version is needed we need to use `UsePythonVersion@0` and/or the trick used in `ci/install_unix.sh`

Reply:
It needs to be the version that matches the pip install of wheel, but otherwise it does not matter. The wheel is a source file and should be cross py2-py3.